### PR TITLE
term check: bloodsucker

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 1]
```diff
@@ -725,6 +725,7 @@ It's certainly something that the planet itself never thought of."
 [sha:6433ad2278c74a017e17800e9d66f5cecafb6092]{
 -- Page 347, Offset 25133.
 -- 真祖である吸血種―――わたしの場合は、そんな形而上の二次的利益を満たしてくれる最上級の食べ物が血液なの。
+// where's the bloodsucker here?
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:728]: 真祖である吸血種 is "bloodsucker true ancestors" so just ignore it.

File: script/Arcueid/Day 10/10_00_ARC10_4.txt [hunk 2]
```diff
@@ -789,7 +790,7 @@ She doesn't know what blood tastes like...?
 [sha:76d67c354f480002545e6a9ef7d9112c35a703e6]{
 -- Page 351, Offset 25145.
 -- 「わたしは吸血種としては半人前だって言ったでしょう。血の味は知らない。それがどんなものなのか、まだわからない。
-"I told you before that I'm a failure of a vampire, right?
+"I told you before that I'm a failure of a bloodsucker, right?
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_4.txt:793]: "as a bloodsucker" might make this a little less awkward. Maybe we want to reconsider how we TL'd 吸血種としては半人前だ

File: script/Arcueid/Day 10/10_00_ARC10_5E.txt [hunk 3]
```diff
@@ -433,6 +433,7 @@ No... perhaps it was more like her fear.
 [sha:4d796d4ec6e3a7f9435712b20b453c3199edd116]{
 -- Page 654, Offset 26015.
 -- 真祖だなんだと偉ぶっておいても所詮は吸血種だな？　同族である吸血鬼を滅ぼして回ろうが、同じ怪物である事に変わりはねえってワケだ！」
+// where to add the bloodsucker here?
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5E.txt:436]: the first "vampire" is "bloodsucker"

File: script/Arcueid/Day 4/04_00_ARC04_6_2C.txt [hunk 4]
```diff
@@ -46,7 +46,7 @@ At some point, her phrasing changed from 'my enemy' to 'our enemy'. I wanted to
 [sha:4a74d676ab54776c0fb03c887c999bf0c51dd85a]{
 -- Page 532, Offset 7010.
 -- 「死徒は人間とか動物とか、とにかく“もともとは吸血種ではなかった”ものたちよ。
-"Dead Apostles are originally things like humans, animals, and anything else that weren't vampires from the start.
+"Dead Apostles are originally things like humans, animals, and anything else that weren't bloodsuckers from the start.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2C.txt:49]: I wonder if you could have a dead apostle mosquito

File: script/Arcueid/Day 4/04_00_ARC04_6_2C.txt [hunk 5]
```diff
@@ -392,6 +392,7 @@ I've only given you a rough outline based on the image you gave me, but do you u
 [sha:a6fbd13b7c80175c995acc62203256e7a4c2e22a]{
 -- Page 561, Offset 7079.
 -- 「乱暴なまとめだけど、とにかく“ヒトのカタチをした、ヒトと同じ知性構造をした吸血種”を吸血鬼と呼ぶの。
+// how to add the bloodsucker here? might need a rephrasing
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6_2C.txt:395]: bloodsuckers are a species

File: script/Arcueid/Day 4/04_00_ARC04_9.txt [hunk 6]
```diff
@@ -106,7 +106,7 @@ But, more importantly―――
 [sha:12c43ca60e58840b9cf91b231d000f5e1e2b3da0]{
 -- Page 694, Offset 7416.
 -- だからいつまでたっても、吸血種としても半人前ってこと」
-That's why no matter what happens, I'll always be a failure of a vampire."
+That's why no matter what happens, I'll always be a failure of a bloodsucker."
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_9.txt:109]: as a bloodsucker

File: script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt [hunk 7]
```diff
@@ -282,6 +282,7 @@ These are upper knights: fully-fledged vampires."
 -- Page 537, Offset 17909.
 -- 完全に“吸血種”として自立。吸血・侵食によって子を作るコトもできるけど、Ⅵ階梯以上の子は作れない。
 // Used offspring for 子, not too sure on this change
+// this is fine as is right?
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt:285]: yes

File: script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt [hunk 8]
```diff
@@ -415,7 +416,8 @@ But if you're trying to produce vampires of rank II and above,
 [sha:fcaa72b383d7c6afe6a0d5cc0cf8ce3eb3759e4b]{
 -- Page 544, Offset 17931.
 -- 「吸血種の血に抗える免疫力……かな？」
-"Something like an immunity against vampire blood, I guess?"
+// bloodsucker blood :aokoheh:
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt:419]: i think make this "vampiric blood". "bloodsucker blood" is a little too ridiculous.

Maybe ask petrikow if ross or yusuke disagree

**Review by Avuxo — APPROVED** [review]
bloodsucker is such a stupid word :sacchincopium:

File: script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt (hunk 8 above)
Comment (yk1512) [reply script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt:419]: I agree

**Review by yk1512 — APPROVED** [review]

**Comment by rschlaikjer** [comment]
Ngl, bloodsucker sounds extremely dumb. Is this a super canon TL thing? [TM wiki](https://typemoon.fandom.com/wiki/Vampire) just says 
> The terms "vampire" and "bloodsucker" are essentially synonymous in all cases except those regarded by the Church as enemies

**Comment by Avuxo** [comment]
So according to that, we should leave "bloodsucker" for the Mario lines? I'm fine getting rid of it entirely personally because "bloodsucker" is a stupid ass term, but that's a topic for the lore council

**Comment by rschlaikjer** [comment]
It seems like actually vampire might be more correct for when the church is speaking? 
> The terms "vampire" and "bloodsucker" are essentially synonymous in all cases except those regarded by the [Church](https://typemoon.fandom.com/wiki/Church) as enemies. They specifically hunt those they deem as "vampires",[[1]](https://typemoon.fandom.com/wiki/Vampire#cite_note-1) [True Ancestors](https://typemoon.fandom.com/wiki/True_Ancestor), [Dead Apostles](https://typemoon.fandom.com/wiki/Dead_Apostle), those turned into bloodsuckers by Dead Apostles, and [mages](https://typemoon.fandom.com/wiki/Magus) that became Dead Apostles through the use of [Magecraft](https://typemoon.fandom.com/wiki/Magecraft), ignoring other bloodsuckers regardless of their species.

Vampires being specifically the ones that the church does target. Dunno whether the wiki is Good in the eyes of the lore council or not, but I would cast my vote for just using vampire.

**Comment by Avuxo** [comment]
From a purely editing perspective, I'll stand with you. If that's what the _wiki_ says, I'd say that probably aligns with what the vast majority of "TM learned" readers are going to understand.

**Comment by yk1512** [comment]
TM Wiki is generally considered to be dogshit in the eyes of the more learned readers. While it is true that the distinction between the two terms isn't used meaningfully in Arc's route, it is important that vampires being a subset of bloodsuckers is established or at least hinted at in Tsukihime early on because Akiha and Sion are both cases where they are bloodsuckers but not vampires.

**Pull request merged**